README.rst

```diff
@@ -0,0 +1,68 @@
+.. raw:: html
+
+    <h1 align="center">
+    <br>
+    <a href="http://duet.openmined.org/"><img src="https://github.com/OpenMined/design-assets/raw/master/logos/OM/mark-primary-trans.png" alt="SyMPC" width="200"></a>
+    <br>
+    SyMPC
+    <br>
+    </h1>
+    <h3 align="center">
+    <br>
+    A library that extends PySyft with SMPC support
+    <br>
+    </h3>
+
+    <a href=""><img src="https://github.com/OpenMined/SyMPC/actions/workflows/tests.yml/badge.svg" /></a>
+    <a href="https://openmined.slack.com/messages/support"><img src="https://img.shields.io/badge/chat-on%20slack-7A5979.svg" /></a>
+    <a href="https://codecov.io/gh/OpenMined/SyMPC"><img src="https://codecov.io/gh/OpenMined/SyMPC/branch/main/graph/badge.svg?token=TS2rZyJRlo" /></a>
+
+
+SyMPC **/ˈsɪmpəθi/** is a library which extends `PySyft <https://github.com/OpenMined/PySyft>`_ ≥0.3 with SMPC support. It allows to compute over encrypted data, to evaluate and to train neural networks.
```

Comment:
```suggestion
SyMPC **/ˈsɪmpəθi/** is a library which extends `PySyft <https://github.com/OpenMined/PySyft>`_ ≥0.3 with SMPC support. It allows computing over encrypted data, and to train and evaluate neural networks.
```

Reply:
Thanks for the feedback @Syzygianinfern0 !